ws-butler.el

```diff
@@ -242,11 +242,11 @@ ensure point doesn't jump due to white space trimming."
      (lambda (_prop beg end)
        (save-excursion
          (setq beg (progn (goto-char beg)
-                          (point-at-bol))
+                          (line-end-position))
```

Comment:
Subtle mistake: `point-at-bol` (note that this is *not* `point-at-eol`!) was changed to `line-end-position`. This causes this function to not actually do anything. I opened #56 with the easy fix.

Reply:
Gah! Sorry for that, but nice catch!